src/clj/fluree/db/query/exec/eval.cljc

```diff
@@ -454,9 +454,9 @@
               (.toString (.getOffset ^OffsetDateTime (->offset-date-time (:value x))))
               #{const/iri-xsd-time}
               (.toString (.getOffset ^OffsetTime (->offset-time (:value x)))))
-       :cljs (.getTimeZoneOffset ^js/Date (if (string? datetime)
-                                            (datatype/coerce (:value datetime) (:datatype-iri datetime))
-                                            datetime)))))
+       :cljs (.getTimeZoneOffset ^js/Date (if (string? x)
+                                            (datatype/coerce (:value x) (:datatype-iri x))
+                                            x)))))
```

Comment:
I think above still has an error in it...

If `x` is a `string?`, then I don't think `(:value x)`, etc. would work.

Should it be `(map? x)`, or maybe `(string? (:value x))`?

Reply:
Good catch, it should be `(string? (:value x))`